src/test/resources/alma-fix/990001412590206441.json

```diff
@@ -74,6 +74,15 @@
     "id" : "http://d-nb.info/gnd/4066724-8"
   } ],
   "subject" : [ {
+    "id" : "120",
```

Comment:
Does this work? https://test.lobid.org/gnd/context.jsonld states id<->@id, and
https://www.w3.org/TR/json-ld/ : "The '@id' keyword means 'This value is an identifier that is an IRI'".
Can "120" act as an IRI? Following https://www.rfc-editor.org/rfc/rfc3987 it can't (needs i.a. a scheme).

Reply:
This was an draft state, it should be now solved.